BadHabbit/BadHabbit/SupportFiles/Fonts/FontName.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  FontName.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 19/07/2023.
+//
+
+import Foundation
+
+enum FontName: String {
```

Comment:
We should remove this file because a similar Fonts.swift file is created by swiftGen automatically.